internal/changelog/renderer.go

```diff
@@ -74,9 +74,9 @@ func (r Renderer) Render() error {
 	tmpl, err := template.New("asciidoc-release-notes").
 		Funcs(template.FuncMap{
 			// nolint:staticcheck // ignoring for now, supports for multiple component is not implemented
-			"linkPRSource": func(component string, id int) string {
+			"linkPRSource": func(component string, id []int) string {
 				component = "agent" // TODO: remove this when implementing support for multiple components
-				return fmt.Sprintf("{%s-pull}%d[#%d]", component, id, id)
+				return fmt.Sprintf("{%s-pull}%v[#%v]", component, id, id)
```

Comment:
This line should produce multiple `{%s-pull}%d[#%d]` strings, while from my understanding this is just printing multiple ints in a single string. Is that correct? If so, please update the logic to create multiple "tags".

Reply:
I've changed the code to return a slice of strings, is this what you meant?

Reply:
I've concatenated the strings with a space.